app/cdap/components/SourceControlManagement/RemotePipelineListView/index.tsx

```diff
@@ -139,6 +181,26 @@ export const RemotePipelineListView = ({ redirectOnSubmit }: IRemotePipelineList
       />
       <PipelineListContainer>
         <SearchBox nameFilter={nameFilter} setNameFilter={setRemoteNameFilter} />
+        {operation && (
+          <Alert
```

Comment:
seems like this is similar to the one in Local pipeline view. nit could be refactor into a common component. But I'm ok as it is now if it's too much work

Reply:
Yes, these table components should be refactored into a common component. Will refactor it in a subsequent commit.